.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: c
+dist: trusty
```

Comment:
Ok,  so that would cut out users running CentOS 6 or lower (or possibly 7.0 or lower?) or Debian Wheezy, right?. That's not necessarily problematic as @Ferroin points out.